nanochat/worker.py

```
@@ -0,0 +1,671 @@
+            gpu.load(args.source, device, model_tag=args.model_tag, step=args.step)
+            iii_client.trigger({"function_id": "state::set", "payload": {
+                "scope": "nanochat:models", "key": "current",
+                "value": {"source": gpu.source, "device": gpu.device,
+                          "config": gpu.meta.get("model_config", {}) if gpu.meta else {}},
+            }})
+        except Exception as e:
+            logger.warn("Auto-load failed, use nanochat.model.load", {"error": str(e)})
+
+    print(f"[nanochat] connected to {args.engine_url}")
+    print(f"[nanochat] model: {'loaded (' + gpu.source + ' on ' + gpu.device + ')' if gpu.ready else 'none'}")
+    print(f"[nanochat] 13 functions, 13 triggers (12 HTTP + 1 queue)")
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)